oneview_redfish_toolkit/conf/redfish.conf

```diff
@@ -12,6 +12,7 @@ ip =
 [credentials]
 userName =
 password =
+authLoginDomain =
```

Comment:
Shouldn't we set "LOCAL" as default?

Reply:
Or does OneView handle no value on this attribute?

Reply:
We can set "LOCAL" as default yes! I had set a empty value just to keep the same default value used for all keys in the `[credentials]` session in that file `redfish.conf` (all values are empty in that session).

Reply:
Ok